content/docs/user-guide/related-technologies.md

```diff
@@ -0,0 +1,135 @@
+  versioned. It leverages Git's features to enable managing different versions
+  of data itself, data pipelines, and experiments.
+
+- DVC is not fundamentally bound to Git, and can work without it (except
+  versioning-related features). This also applies to Git-LFS and Git-annex,
+  below.
+
+### Git-LFS (Large File Storage)
+
+- DVC does not require special servers like Git-LFS demands. Any cloud storage
+  like S3, Google Cloud Storage, or even an SSH server can be used as a
+  [remote storage](/doc/command-reference/remote). No additional databases,
+  servers, or infrastructure are required.
+
+- DVC does not add any hooks to the Git repo by default (although they are
+  [available](/doc/command-reference/install)).
+
+- Git-LFS was not made with data science in mind, so it doesn't provide related
+  features (e.g. [pipelines](/doc/user-guide/basic-concepts#data-pipeline),
+  [metrics](/doc/command-reference/metrics), etc.).
+
+- Github (most common Git hosting service) has a limit of 2 GB per repository.
+
+### Git-annex
+
+- DVC can use reflinks\* or hardlinks (depending on the system) instead of
+  symlinks to improve performance and the user experience.
+
+- Git-annex is a datafile-centric system whereas DVC focuses on providing a
+  workflow for machine learning and reproducible experiments. When a DVC or
+  Git-annex repository is cloned via `git clone`, data files won't be copied to
+  the local machine, as file contents are stored in separate
+  [remotes](/doc/command-reference/remote). With DVC however, `.dvc` files,
+  which provide the reproducible workflow, are always included in the Git
+  repository. Hence, they can be executed locally with minimal effort.
+
+- DVC optimizes file hash calculation.
+
+> \* **copy-on-write links or "reflinks"** are a relatively new way to link
+> files in UNIX-style file systems. Unlike hardlinks or symlinks, they support
+> transparent [copy on write](https://en.wikipedia.org/wiki/Copy-on-write). This
+> means that editing a reflinked file is always safe as all the other links to
+> the file will reflect the changes.
+
+### Git workflows/methodologies such as Gitflow
+
+- DVC enables a new experimentation methodology that integrates easily with
+  existing Git workflows. For example, a separate branch can be created for each
+  experiment, with a subsequent merge of the branch if the experiment is
+  successful.
+
+- DVC innovates by giving users the ability to easily navigate through past
+  experiments without recomputing them each time.
+
+### Workflow management tools
+
+Pipelines and dependency graphs
+([DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) such as _Airflow_,
+_Luigi_, etc.
+
+- DVC is focused on data science and modeling. As a result, DVC pipelines are
+  lightweight and easy to create and modify. However, DVC lacks advanced
+  pipeline execution features like execution monitoring, error handling, and
+  recovering.
```

Comment:
> DVC lacks advanced pipeline execution features...

Is this still valid and do we want to highlight it?

Reply:
yes, it is true ... no need to highlight